docs/data/material/guides/minimizing-bundle-size/minimizing-bundle-size.md

```diff
@@ -207,6 +207,57 @@ It will perform the following diffs:
 +import { Button, TextField } from '@mui/material';
 ```
 
+### Option three: use a SWC plugin
+
+This option pretty much similar to the option two, except for the developer who is using [SWC Compiler](https://swc.rs/). Use this [plugin](https://github.com/swc-project/plugins/blob/main/packages/transform-imports/README.md):
+
+`yarn add -D @swc/plugin-transform-imports`
+
+Modify your .swcrc or Webpack configuration:
```

Comment:
Below we are showing the config for Webpack, so mentioning `.swcrc` here may confuse users. It would be better to show a separate config for `.swcrc`.
```suggestion
Modify your webpack configuration as shown below:
```